pybop/optimisers/base_optimiser.py

```diff
@@ -61,19 +62,18 @@ def __init__(
         self.parameters = Parameters()
         self.x0 = None
         self.bounds = None
-        self.sigma0 = 0.1
+        self.sigma0 = 0.02
```

Comment:
Was there a specific condition that required this change? Shrinking the default `sigma0` reduces the explored space for the stochastic methods, which could lead to more local optimums for users that are not adjusting this value.

Reply:
The default value of 0.02 for just one of the Pints classes was not implemented correctly (the expected value was overwritten). The simplest change was to have all optimisers use the same default value.